.tours/add-a-new-scraper.tour

```diff
@@ -0,0 +1,609 @@
+      "selection": {
+        "start": {
+          "line": 10,
+          "character": 13
+        },
+        "end": {
+          "line": 10,
+          "character": 48
+        }
+      },
+      "description": "### Map Config Properties\n\nAdd a call to `Map()` for each property you need to deserialize. The deserialization code automatically looks for Yaml fields with the same name as your property, but starting with a lowercase letter. For example, the `QueueName` property can be deserialized from this Yaml:\n\n```yaml\nqueueName: 'promitor-queue'`\n```"
+    },
+    {
+      "file": "src/Promitor.Core.Scraping/Configuration/Serialization/v1/Providers/ServiceBusQueueDeserializer.cs",
+      "selection": {
+        "start": {
+          "line": 11,
+          "character": 17
+        },
+        "end": {
+          "line": 11,
+          "character": 30
+        }
+      },
+      "description": "### Identify Required Properties\n\nProperties are optional unless marked as required. If a property is marked as required, Promitor will display a validation error to the user if it isn't provided."
+    },
+    {
+      "file": "src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/AzureMetadataDeserializer.cs",
+      "selection": {
+        "start": {
+          "line": 20,
+          "character": 17
+        },
+        "end": {
+          "line": 20,
+          "character": 64
+        }
+      },
+      "description": "### Specify Default Values\n\nIf a field needs to have a certain default value when not specified, and the default isn't the default value for that type of object in .NET, you can provide an alternative default using `WithDefault()`."
+    },
+    {
+      "file": "src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/AzureMetadataDeserializer.cs",
+      "selection": {
+        "start": {
+          "line": 21,
+          "character": 17
+        },
+        "end": {
+          "line": 21,
+          "character": 47
+        }
+      },
+      "description": "### Custom Mapper\n\nIf you need to provide custom logic to deserialize a particular field you can do that with `MapUsing()`. This allows you to customise the deserialization process without using a child deserializer."
+    },
+    {
+      "file": "src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/MetricDefaultsDeserializer.cs",
+      "selection": {
+        "start": {
+          "line": 14,
+          "character": 17
+        },
+        "end": {
+          "line": 14,
+          "character": 63
+        }
+      },
+      "description": "### Using a Child Deserializer\n\nIf you need to deserialize a complex property rather than a simple field, you can tell the deserialization framework to use a child validator to perform the deserialization. You can pass a parameter of type `IDeserializer<TChild>` into your constructor, and the Dependency Injection framework will automatically provide it for you."
```

Comment:
Maybe this step is relevant for adding a new scraper?

Reply:
Just checking - do you mean "not relevant"? If so, yeah, I think we could potentially drop some of these steps. Happy to get rid of it - I'll just wait to double check that's what you meant.

Reply:
Yes but it's ok - We can still remove later on!